base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -136,11 +138,17 @@ pub enum OutputManagerRequest {
     CreateClaimShaAtomicSwapTransaction(HashOutput, PublicKey, MicroTari),
     CreateHtlcRefundTransaction(HashOutput, MicroTari),
     GetOutputStatusesByTxId(TxId),
-    GetNextSpendAndScriptKeys,
-    GetRecoveryData,
+    GetNextSpendAndScriptKeys, // TODO: Remove this when core key manager is implemented
+    GetNextSpendAndScriptKeyIds,
+    GetRecoveryData, // TODO: Remove this when core key manager is implemented
+    GetRecoveryKeyId,
+    TryCommitmentKeyRecovery(Commitment, EncryptedData, Option<KeyId>),
+    GetSpendingKeyId(PublicKey),
+    GetDiffieHellmanSharedSecret(KeyId, PublicKey),
```

Comment:
we can remove all of these we don't need them

Reply:
I will verify and remove, together with their downline implementations

Reply:
I have removed `GetNextSpendAndScriptKeys`, `GetRecoveryData` and `GetDiffieHellmanSharedSecret` for now